src/templates/shared/wrapper.js

```diff
@@ -0,0 +1,86 @@
+const {html, alert, ol, pageAnchor} = require("./bits");
+
+const DISCORD_URL = "https://discord.reclaimers.net";
+const REPO_URL = "https://github.com/Sigmmma/c20";
+
+const STUB_ALERT = alert({type: "danger", body: html`
+  <p>🚧 This article is a stub. You can help expand it by submitting content in
+  pull requests or issues in this wiki's <a href="${REPO_URL}">source repo</a>.</p>
+`});
+
+const wrapper = (page, metaIndex, body) => {
+  const srcUrl = `${REPO_URL}/tree/master/src/content${page._dirUrl}`;
+  const breadcrumbs = [];
+  for (let i = 0; i <= page._dir.length; i++) {
+    const parentUrl = "/" + page._dir.slice(0, i).join("/");
+    const parent = metaIndex.pages.find(otherPage => otherPage._dirUrl == parentUrl);
+    if (parent) {
+      breadcrumbs.push(parent);
+    }
+  }
+  return html`
+    <!DOCTYPE html>
+    <html lang="en">
+      <head>
+        <meta charset="utf-8"/>
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <meta http-equiv="X-UA-Compatible" content="ie=edge">
+        <meta property="og:title" content="${page.title}"/>
+        <meta property="og:site_name" content="The Reclaimers Library"/>
+        <meta property="og:type" content="website"/>
+        <meta property="og:locale" content="en_US"/>
+        <meta property="og:url" content="${metaIndex.baseUrl}${page._dirUrl}"/>
```

Comment:
These `og:` properties determine how the page is previewed when shared on sites like Facebook or within Discord channels.